BUILD.bazel

```diff
@@ -1,14 +1,6 @@
-_msvc_copts = ["/std:c++17"]	
-_gcc_copts = ["-std=c++17"]
+package(default_visibility = ["//visibility:public"])
 
-cc_library(
+alias(
     name = "entt",
-    visibility = ["//visibility:public"],
-    strip_include_prefix = "src",
```

Comment:
I moved this target to the `src` directory to avoid using `strip_include_prefix`. There are issues with the virtual file paths when generating compile_commands.json for editor integration so I like to avoid it if possible.

Reply:
I barely get what you mean 😅 I guess I'll trust you here. 👍